lib/components/Upload.jsx

```diff
@@ -204,16 +204,7 @@ var Upload = React.createClass({
     }
     if (this.isUploadFailed()) {
       var uploadError = this.getUploadError();
-
-      if (getIn(uploadError, ['error', 'code']) && getIn(uploadError, ['error', 'message'])) {
-          return <div className="Upload-status Upload-status--error">{uploadError.error.message}</div>;
-      }
-
-      if (uploadError.code && uploadError.code === 404) {
-        return <div className="Upload-status Upload-status--error">{uploadError.message}</div>;
-      }
-
-      return <div className="Upload-status Upload-status--error">{'The upload didn\'t work.'}</div>;
+      return <div className="Upload-status Upload-status--error">{uploadError.message}</div>;
```

Comment:
yeah - and the top level I have split it into the `message` and also `debug`. The debug info contains all the extras such as the UTC datetime. Yeah so I want get design input so we dislapy all the info but it is split into the friendly `message` and the more detailed and helpful to us `debug` details